.github/workflows/downstream_protobuf_compatibility_check_nightly.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  pull_request:
+    # Updates on Release-Please PRs
+    branches:
+      - 'release-please--branches--main'
```

Comment:
I thought this means this workflow would run on any PRs targeting 'release-please--branches--main', not PRs created from 'release-please--branches--main', can you verify it?

Reply:
Oh, I see. Yeah, I think you're right. It's for the branch that it's targeting and not where it is raised from.

I think there are other (not built in) solutions for this.

Reply:
Added a conditional that checks the head_ref (source branch) and the github.event type (schedule) so that it works for release please PRs and the nightly flow